js/ui/form/ui.form.js

```diff
@@ -635,6 +635,60 @@ var Form = Widget.inherit({
              * @type number
              * @default undefined
              */
+            /**
+            * @name dxFormButtonItem
+            * @publicName Button
+            * @section FormItems
+            * @type object
+            */
+            /**
+             * @name dxFormButtonItem_name
+             * @publicName name
+             * @type string
+             * @default undefined
+             */
+            /**
+             * @name dxFormButtonItem_colSpan
+             * @publicName colSpan
+             * @type number
+             * @default undefined
+             */
+            /**
+             * @name dxFormButtonItem_itemType
+             * @publicName itemType
+             * @type Enums.FormItemType
```

Comment:
Do we need to add the "button" item to the "Enums.FormItemType" enum?

Reply:
Yes, I'll fix it